README.md

```diff
@@ -47,7 +47,7 @@ To see current date & time in a format ready to be put into the file run
 `date --utc --rfc-3339=seconds`.
 
 - New, not yet resolved issue or announcement about future maintenance
-  - set `resolved: false`
+  - remove `resolved:`
```

Comment:
I think we want `resolved: false` in case of not yet resolved issue and remove it in case of an announcement?

Reply:
Oh, right, I didn't realize that this paragraph is also about unresolved new issues